web/src/styles/scss/print.scss

```diff
@@ -0,0 +1,54 @@
+@media print {
```

Comment:
Copied these from `print.css`, which is deleted.

Reply:
Aha! I wondered what was going on there. Thanks for moving this over!